### DISCUSSION TITLE
Flatpak

### DISCUSSION THREAD

**Discussion by Darin755** [coreemu/core#885] [Ideas]
Would it be possible to package this as a flatpak on Flathub? I know the sandbox could make things tricky.

**Comment by bharnden** [comment]
Possibly, would be something to look into and consider for the future. There is the current path of being able to run CORE from Docker as a short term, somewhat similar concept.

**Reply by Darin755** [reply]
I think Flatpak is much cleaner overall and it has the benefit of being well known on the desktop. Just something to ponder.
